worker/pkg/workflows/datasync/activities/post-table-sync/activity.go

```diff
@@ -0,0 +1,133 @@
+package posttablesync_activity
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log/slog"
+
+	"connectrpc.com/connect"
+	mgmtv1alpha1 "github.com/nucleuscloud/neosync/backend/gen/go/protos/mgmt/v1alpha1"
+	"github.com/nucleuscloud/neosync/backend/gen/go/protos/mgmt/v1alpha1/mgmtv1alpha1connect"
+	neosynclogger "github.com/nucleuscloud/neosync/backend/pkg/logger"
+	"github.com/nucleuscloud/neosync/backend/pkg/sqlmanager"
+	"github.com/nucleuscloud/neosync/worker/pkg/workflows/datasync/activities/shared"
+	"go.temporal.io/sdk/activity"
+	"go.temporal.io/sdk/log"
+)
+
+type Activity struct {
+	jobclient        mgmtv1alpha1connect.JobServiceClient
+	sqlmanagerclient sqlmanager.SqlManagerClient
+	connclient       mgmtv1alpha1connect.ConnectionServiceClient
+}
+
+func New(
+	jobclient mgmtv1alpha1connect.JobServiceClient,
+	sqlmanagerclient sqlmanager.SqlManagerClient,
+	connclient mgmtv1alpha1connect.ConnectionServiceClient,
+) *Activity {
+	return &Activity{
+		jobclient:        jobclient,
+		sqlmanagerclient: sqlmanagerclient,
+		connclient:       connclient,
+	}
+}
+
+type RunPostTableSyncRequest struct {
+	JobId string
+	// Identifier that is used in combination with the AccountId to retrieve the post table run config
+	Name      string
+	AccountId string
+}
+type RunPostTableSyncResponse struct {
+}
+
+func (a *Activity) RunPostTableSync(
+	ctx context.Context,
+	req *RunPostTableSyncRequest,
+) (*RunPostTableSyncResponse, error) {
+	activityInfo := activity.GetInfo(ctx)
+	loggerKeyVals := []any{
+		"jobId", req.JobId,
+		"WorkflowID", activityInfo.WorkflowExecution.ID,
+		"RunID", activityInfo.WorkflowExecution.RunID,
+	}
+	logger := log.With(
+		activity.GetLogger(ctx),
+		loggerKeyVals...,
+	)
+	logger.Debug("running post table sync activity")
+	slogger := neosynclogger.NewJsonSLogger().With(loggerKeyVals...)
+
+	jobResp, err := a.jobclient.GetJob(ctx, connect.NewRequest(&mgmtv1alpha1.GetJobRequest{Id: req.JobId}))
```

Comment:
just curious but since you are retrieving the job here, do you need to pass in the account id to the request? You can probably get away with not having it since you are retrieving the job anyways.